website/docs/learning-and-resources/sample-apps.md

```diff
@@ -49,7 +49,7 @@ import TabItem from '@theme/TabItem';
            <strong>Server-side Search</strong>
         </div><hr/>
```

Comment:
I don't think you have updated this - 
* Show data in table - takes you to the [Pagination](https://app.appsmith.com/applications/623cca594d9aea1b062b33c6/pages/623cca594d9aea1b062b33cd) app.
* External Filters for Table (View App1) - takes you to [Conditionally show data](https://app.appsmith.com/applications/65420e152cf3bf5c05cd011f/pages/65420e152cf3bf5c05cd0134)
* Show data from multiple queries - takes you to [Conditionally Change Table Data Based on queryParams](https://app.appsmith.com/applications/65420e152cf3bf5c05cd011f/pages/65420e152cf3bf5c05cd0138)
* Add List Options Dynamically - takes you to [Server side filter to a table](https://app.appsmith.com/applications/61fbdf232cd3d95ca414b805/pages/61fbdf232cd3d95ca414b808)
* Graphql - Fetch Data - it takes you to [GraphQL pagination ](https://app.appsmith.com/applications/61f81d4f9d6d6a6720c9b4c7/pages/62bddc01612750023eaff6d3)and also seems to be broken.